eva_submission/vep_utils.py

```diff
@@ -283,3 +292,22 @@ def download_and_extract_vep_cache(ftp, vep_cache_file, taxonomy_id):
     os.makedirs(os.path.join(cfg['vep_cache_path'], species_name), exist_ok=True)
     shutil.move(sources[0], copy_destination)
     tmp_dir.cleanup()
+
+
+@retry(tries=4, delay=2, backoff=1.2, jitter=(1, 3), logger=logger)
+def download_and_install_vep_version(vep_version):
+    file_download_url = f'https://github.com/Ensembl/ensembl-vep/archive/release/{vep_version}.zip'
+
+    # Download the Vep version
+    tmp_dir = tempfile.TemporaryDirectory()
+    destination = os.path.join(tmp_dir.name, f'{vep_version}.zip')
+    wget.download(file_download_url, destination)
```

Comment:
Do we need to add another dependency ? 
Would something like [urlretrieve](https://docs.python.org/3/library/urllib.request.html#urllib.request.urlretrieve) work ?

Reply:
updated to use requests library